docarray/array/doc_vec/column_storage.py

```diff
@@ -63,12 +64,22 @@ def __len__(self) -> int:
     def __getitem__(self: T, item: IndexIterType) -> T:
         if isinstance(item, tuple):
             item = list(item)
-        tensor_columns = {key: col[item] for key, col in self.tensor_columns.items()}
-        doc_columns = {key: col[item] for key, col in self.doc_columns.items()}
+        tensor_columns = {
+            key: col[item] if col is not None else col
```

Comment:
```suggestion
            key: col[item] if col is not None else None
```
It's the same, but a tad more readable imo, up to you if you want to do it this way :)
same for the lines below

Reply:
fair